open-hackathon-server/src/hackathon/constants.py

```diff
@@ -296,9 +296,16 @@ class FILE_TYPE:
 
     File of different types may saved in different position
     """
+    AZURE_CERT = "azure_cert"
     TEMPLATE = "template"
     HACK_IMAGE = "hack_image"
-    AZURE_CERT = "azure_cert"
+    USER_AVATAR = "user_avatar"
```

Comment:
please make the type "rougher". Classify the storage in higher level rather than defines a type for every feature